polaris-react/src/components/AppProvider/AppProvider.tsx

```diff
@@ -24,6 +24,39 @@ import type {FeaturesConfig} from '../../utilities/features';
 import './AppProvider.scss';
 import './global.scss';
 
+function measureScrollbars() {
+  const maxScrollbarWidth = 20;
+  const overflowParentHeight = 30;
+  const overflowChildHeight = 40;
```

Comment:
nit: looking at this again, I'm think maybe `CHILD_HEIGHT = PARENT_HEIGHT + 10` rather than just do the math ourselves. I think this makes it more clear what we're trying to achieve with these constants

Reply:
Done!